requirements/DATA_STORE_AURORA_0006/testing.md

```diff
@@ -0,0 +1,11 @@
+# Testing DATA_STORE_AURORA_0006
+
+**1. Steps:**
+
+* Build the project with "make" command
+* Run ./work/binaries/demo file
+
+**2. Expected results:**
+
+* Project should succesfully build
+* The running binary should output logs in a loop with item reception messages
```

Comment:
requirement is  The Data Interface shall provide the capability to delete data in memory
please add an example output

Reply:
Done